includes/admin/class-wcs-admin-post-types.php

```diff
@@ -200,12 +205,14 @@ private function posts_clauses_high_performance( $pieces ) {
 
 	/**
 	 * Displays the dropdown for the product filter
+	 *
+	 * @param string $order_type The type of order. This will be 'shop_subscription' for Subscriptions.
+	 *
 	 * @return string the html dropdown element
 	 */
-	public function restrict_by_product() {
-		global $typenow;
+	public function restrict_by_product( $order_type ) {
```

Comment:
We may have to deprecate these non-private functions in case 3PDs are using them.

I'm unsure about this though, as we're adding an argument rather than removing or modifying the argument type.

What do you think?

Reply:
Similar situation for `restrict_by_payment_method` and `restrict_by_customer` functions